src/rendering/screenspacerenderableframebuffer.cpp

```diff
@@ -68,35 +67,37 @@ ScreenSpaceFramebuffer::ScreenSpaceFramebuffer(const ghoul::Dictionary& dictiona
         iIdentifier = id();
```

Comment:
Nitpicky, but this variable name was confusing to me, and the variable could be initialized in the if-statement instead of outside :)  Maybe:

```suggestion
        int framebufferIdCounter= id();
```
?